### ISSUE TITLE
Kein Außentemperaturfühler seit sensoApp

### ISSUE THREAD

**Issue by Michi0606** [thomasgermain/vaillant-component#242]
Since i swiched to new Option "senso App" there is no Sensor for Außentemperaturfühler any more.

Best Regards 
Michael

**Comment by thomasgermain** [comment]
Hello,

Can you confirm "Außentemperaturfühler" is the target high temperature?

**Comment by Michi0606** [comment]
No, its temperature sensor for outside temperature (outside the house)

Best Regards
Michael

**Comment by thomasgermain** [comment]
@sylvainvh @xaviergriffon, before digging into the code, do you have outdoor temperature sensor in your system ?

**Comment by sylvainvh** [comment]
Hi @thomasgermain ,

I've an outdoor temperature sensor and it integration works perfectly.
![Screenshot_20231025-073240](https://github.com/thomasgermain/vaillant-component/assets/5175164/572941e6-01ea-4b34-bd19-53f845d4c89c)

**Comment by xaviergriffon** [comment]
> @sylvainvh @xaviergriffon, before digging into the code, do you have outdoor temperature sensor in your system ? 

Hi,

The outside temperature also works for me.

**Comment by thomasgermain** [comment]
Thanks ;)

@Michi0606 do you have any error in the log while setuping the integration ?

**Comment by Michi0606** [comment]
Hello @thomasgermain 
i dont have logs from integration setup any more. all i can say that i had outside temperature sensor in the "old" version before switching to sensoApp-Option.

![image](https://github.com/thomasgermain/vaillant-component/assets/128812272/956a0d3b-75bc-4603-9b96-d08ea005519b)

![image](https://github.com/thomasgermain/vaillant-component/assets/128812272/a5936874-2fce-428b-92d4-108f413ca043)


best regards

**Comment by Krzysztonek** [comment]
@Michi0606, I see some more entities in my integration than you have on your screenshot:
![obraz](https://github.com/thomasgermain/vaillant-component/assets/87784133/cf7c051d-c202-45ee-9780-656a69c17a21)
@thomasgermain, does it mean that I should reinstall my integration? Where does such difference come from if the list of my devices is exactly the same as @Michi0606 shows above?

**Comment by thomasgermain** [comment]
It could come from the setup. This is why I'm asking for the logs :).

The vaillant API is quite unstable, so sometimes, during the setup, API returns an error for a sensor, so this sensor is never added in HA (and thus never refreshed). In that case, the only way to get new sensors is to reload the integration (or restart HA)
